.ci/run_e2e_tests.sh

```diff
@@ -0,0 +1,16 @@
+SCRIPTS_PATH=$(cd $(dirname "${BASH_SOURCE[0]}"); pwd)
+SRC_PATH=$(dirname "${SCRIPTS_PATH}")
+# Build the runnable Gecko docker image
+bash "${SRC_PATH}"/scripts/build_image.sh
+GECKO_IMAGE=$(docker image ls --format="{{.Repository}}" | head -n 1)
+
+# Turn off GO111MODULE to pull e2e test source code in order to get run script.
+GO111MODULE=off go get -t -v github.com/kurtosis-tech/ava-e2e-tests/...
```

Comment:
I am not sure I understand why we need to switch off `GO111MODULE` here.

Reply:
@holisticode in go 1.13, with GO111MODULE=on, `go get` will not pull source code. ( https://github.com/golang/go/issues/31529 ) We have scripts that build necessary docker images which interact with the network, so we want to pull the source from the test repo to build those images.